tests/test_manager.py

```diff
@@ -43,6 +43,18 @@ def test_profiling_log():
     mock_path.read_text.assert_called_once()
 
 
+def test_profiling_experiment():
+    """Test the ProfilingExperiment class."""
+
+    experiment = ProfilingExperiment(path=Path("/fake/path"))
+
+    assert experiment.path == Path("/fake/path")
+    assert experiment.status == ProfilingExperimentStatus.NEW
+
+    experiment.status = ProfilingExperimentStatus.RUNNING
+    assert experiment.status == ProfilingExperimentStatus.RUNNING
```

Comment:
If this test failed, I would start questioning reality (but I can see how this test will catch any accidental changes the setter method)